src/main/java/io/cdap/plugin/ariba/source/AribaServices.java

```diff
@@ -663,17 +690,9 @@ public JsonNode fetchData(String jobId, String fileName)
 
     OkHttpClient enhancedOkHttpClient = getConfiguredClient().build();
```

Comment:
this line can be removed as it is not used.

Reply:
Removed !